composer.json

```diff
@@ -13,16 +13,29 @@
     "MIT"
   ],
   "require" : {
-    "php": "^5.5.9|>=7.0.8",
-    "symfony/framework-bundle": "~3.4|~4.0|~5.0",
-    "symfony/validator": "~3.4|~4.0|~5.0",
-    "lexik/jwt-authentication-bundle": "^1.1|^2.0@dev"
+    "php": ">=7.1.3",
+    "lexik/jwt-authentication-bundle": "^1.1|^2.0",
+    "symfony/config": "^3.4|^4.4|^5.2",
+    "symfony/console": "^3.4|^4.4|^5.2",
+    "symfony/dependency-injection": "^3.4.27|^4.4|^5.2",
```

Comment:
The `spec\Gesdinet\JWTRefreshTokenBundle\DependencyInjection\GesdinetJWTRefreshTokenExtensionSpec` class makes use of the internal `ContainerBuilder::removeBindings()`, which was renamed at 3.4.27 from whatever it was previously, hence the minimum here being set on a later patch release.